jobs/fetch-universe-metadata/TimerFunction/index.py

```diff
@@ -19,6 +22,27 @@ def main(mytimer: func.TimerRequest, universe="epiverse-connect"):
             - Source repository URL
             - List of article URLs
     """
+
+    connection_str = os.getenv("AzureWebJobsStorage")
+    connection_string = os.getenv("AzureWebJobsStorage")
+    if connection_string:
+        logging.INFO(f"AzureWebJobsStorage: {connection_string}")
+    else:
+        logging.ERROR("AzureWebJobsStorage is not set!")
```

Comment:
Can we use a single variable for this?

```suggestion
    connection_str = os.getenv("AzureWebJobsStorage")
    if connection_str:
        logging.INFO(f"AzureWebJobsStorage: {connection_str}")
    else:
        logging.ERROR("AzureWebJobsStorage is not set!")
```

Reply:
Fixed in [`b41827b` (#20)](https://github.com/epiverse-connect/epiverse-search-backend/pull/20/commits/b41827b5217b59312deb475ad0f161f391ef601c).